armi/reactor/reactors.py

```diff
@@ -2269,6 +2269,19 @@ def processLoading(self, cs, dbLoad: bool = False):
                 )[0]
                 for a in self.parent.blueprints.assemblies.values():
                     a.makeAxialSnapList(refAssem=finestAssemblyMesh)
+            if not cs["inputHeightsConsideredHot"]:
+                self._axiallyExpandAssems(
+                    self.parent.blueprints.assemblies.values(),
+                    dbLoad,
+                    "blueprints",
+                    cs["detailedAxialExpansion"],
+                )
+                if not cs["detailedAxialExpansion"]:
+                    refMesh = (
+                        self.refAssem.getAxialMesh()
```

Comment:
it mgiht be good to use finestAssemblyMesh

Reply:
actually, we need to use finestAssemblyMesh. if we don't it will over expand the blueprints to the irradiate fuel core height.  We shoudl probably rename
finestAssemblyMesh
to 
finestMeshAssembly

Reply:
> we need to use finestAssemblyMesh

I'm not really sure it matters. When an assembly gets created from the blueprint assemblies, it gets created in `reactors.py::Core::createAssemblyOfType`. In this method, if `detailedAxialExpansion: False`, `setBlockMesh` is run. So whichever mesh `setBlockMesh` uses here (in Core::processLoading) just gets overwritten in `createAssemblyOfType` (which happens to be `core.p.referenceBlockAxialMesh()`.